### DISCUSSION TITLE
part-ii/2017-139/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#264] [Question comments]
Part II 2017 Statistical Physics: Paper 2, Section II, 34D 

https://questions.tripos.org/part-ii/2017-139/

**Comment by thenewtonium** [comment]
This question is basically entirely bookwork.

<details><summary>Some points about (b)(iii)</summary>

Considering the (constant) ratio $S/N$ is helpful since things cancel out to it being a function of $\mu /T$ only, which is the intended way to show it is constant. (Initally I got the powers of $T$ wrong so didn't spot this and differentiated instead, which works but is silly in this case).

Constant $V T^{c}$ is obtained from noting that $N = \frac{1}{k_B T} \frac{\partial}{\partial \mu} \log \mathcal{Z}$ is constant (so $c = b = a+1$).

Constant $pV^d$ is obtained by calculating $p = -\frac{\partial \Phi}{\partial V} = k_B T \frac{\partial}{\partial V} \log \mathcal{Z}$, factoring out (constant) $(VT^b)^{\frac{b+1}{b}}$ and rearranging to obtain that $pV^{\frac{b+1}{b}}$ is constant, i.e. $d= \frac{a+2}{a+1}$.

</details>
